### ISSUE TITLE
anacron: ignores -m command line parameter

### ISSUE THREAD

**Issue by corvus1** [cronie-crond/cronie#133]
Not sure if intentional or not, but inconsistent. If you pass `-m` to the cron daemon, it's using alternative MTA command to mail the output of the cronjobs. Works fine with crond, and I think it's the standard for any crond implementation. Anacron, however, still calls `/usr/sbin/sendamail`. 

Can it be corrected somehow?

**Comment by t8m** [comment]
Anacron does not have any way to change the mailer command. It would be a new feature.

**Comment by corvus1** [comment]
Can I humbly request it?

**Comment by corvus1** [comment]
> > Can I humbly request it?
> 
> There is no active development of cronie going on right now. But I review and merge external contributions if there are any.

So you're saying if I do it myself, there's a chance of getting a pull request merged? 
I am... not great at C, but I might try...

**Comment by corvus1** [comment]
My first idea was to read a variable from anacrontab. It already has MAILTO and MAILFROM, I was thinking of adding something like MAILER. Or even just SENDMAIL. And stuffing it here: 
https://github.com/cronie-crond/cronie/blob/557850377992456c025e2efa0ec4cd3e90c52d83/anacron/global.h#L36
Somehow it already has a variable defined. Says it's user-configurable... Or is it configurable at build-time?

**Comment by t8m** [comment]
> Says it's user-configurable... Or is it configurable at build-time?

Yeah, it is meant to be configurable by editing the global.h file. Which is of course build-time and not particularly friendly way.

**Comment by corvus1** [comment]
Huh, but, being a gentoo-user, might be fine for me. :-) But okay. 

Then if we look here:
https://github.com/cronie-crond/cronie/blob/557850377992456c025e2efa0ec4cd3e90c52d83/anacron/runjob.c#L316-L328

We're reading variables from the environment. I'll read another one, SENDMAIL, in the same way as other 2. And if it's not defined in the environment, it'll be left as defined in global.h, right? Sounds pretty simple to implement. I might even be able to do it, without too much trouble. And then just to document it in the man page. 

Does that sound sane? I'm asking all these questions, because I know how easy it is to make a horrible mess in C.

**Comment by t8m** [comment]
> Does that sound sane? I'm asking all these questions, because I know how easy it is to make a horrible mess in C.

Yeah, that would be fine.

**Comment by corvus1** [comment]
Here's what I managed so far. https://github.com/cronie-crond/cronie/pull/135

**Comment by corvus1** [comment]
@t8m Would you care to take a quick look at my draft?
